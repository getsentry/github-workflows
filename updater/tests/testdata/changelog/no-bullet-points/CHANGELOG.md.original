# Changelog

## 0.14.0

### Dependencies

This section contains only plain text with no bullet points.
The update-changelog script should handle this case gracefully.
