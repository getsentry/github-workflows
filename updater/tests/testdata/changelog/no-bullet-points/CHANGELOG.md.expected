# Changelog

## Unreleased

### Dependencies

- Bump Dependency from v7.16.0 to v7.17.0 ([#123](https://github.com/getsentry/dependant/pulls/123))
  - [changelog](https://github.com/getsentry/dependency/blob/main/CHANGELOG.md#7170)
  - [diff](https://github.com/getsentry/dependency/compare/7.16.0...7.17.0)

## 0.14.0

### Dependencies

This section contains only plain text with no bullet points.
The update-changelog script should handle this case gracefully.